data/crac/crac-impl/src/test/java/com/powsybl/openrao/data/cracimpl/utils/ExhaustiveCracCreation.java

```diff
@@ -290,7 +290,7 @@ public static Crac create(CracFactory cracFactory) {
                 .newRange().withMin(-1000).withMax(1000).add()
                 .newOnContingencyStateUsageRule().withContingency("contingency1Id").withInstant(CURATIVE_INSTANT_ID).withUsageMethod(UsageMethod.AVAILABLE).add()
                 .newOnContingencyStateUsageRule().withContingency("contingency2Id").withInstant(CURATIVE_INSTANT_ID).withUsageMethod(UsageMethod.AVAILABLE).add()
-                .newOnFlowConstraintUsageRule().withInstant(PREVENTIVE_INSTANT_ID).withFlowCnec("cnec3curId").withUsageMethod(UsageMethod.AVAILABLE).add()
```

Comment:
could you please reformat the whole class to fix continuation indentation, thanks :)